doc/source/user_guide/geometrical_transform.rst

```diff
@@ -0,0 +1,132 @@
+============================================
+Geometrical transformations of images
+============================================
+
+Cropping, resizing and rescaling images
+---------------------------------------
+
+.. currentmodule:: skimage.transform
+
+Images being NumPy arrays (as described in the :ref:`numpy` section), cropping
+an image can be done with simple slicing operations. Below we crop a 100x100
+square corresponding to the top left corner of the astronaut image. Note that
+this operation is done for all color channels (the color dimension is the last
+third dimension):: 
+
+   >>> from skimage import data
+   >>> img = data.astronaut()
+   >>> top_left = img[:100, :100]
+
+In order to change the shape of the image, :mod:`skimage.color` provides several
+functions described in :ref:`sphx_glr_auto_examples_transform_plot_rescale.py`
+.
+
+.. literalinclude:: ../../examples/transform/plot_rescale.py
+    :language: python
+    :start-after: import matplotlib.pyplot as plt
+    :end-before: fig, axes
+
+
+.. image:: ../auto_examples/transform/images/sphx_glr_plot_rescale_001.png
+   :target: ../auto_examples/transform/plot_rescale.html
+   :align: center
+   :width: 80%
+
+Projective transforms (homographies)
+------------------------------------
+
+`Homographies <https://en.wikipedia.org/wiki/Homography>`_
+are transformations of a Euclidean space that preserve the alignment of points.
+Specific cases of homographies correspond to the conservation of more properties,
+such as parallelism (affine transformation), shape (similar transformation) or distances (Euclidean transformation).
+The different types of homographies available in scikit-image are presented in 
+:ref:`sphx_glr_auto_examples_transform_plot_transform_types.py`.
+
+Projective transformations can either be created using the explicit
+parameters (e.g. scale, shear, rotation and translation) or the full
+transformation matrix::
```

Comment:
If you mention the transformation matrix, you should add example code with the matrix, as well as mention that it uses homogeneous coordinates.

Reply:
done, are you ok with the result?